test/Microsoft.ML.Functional.Tests/Debugging.cs

```diff
@@ -105,7 +105,6 @@ public void InspectPipelineSchema()
 
             // Define a pipeline
             var pipeline = mlContext.Transforms.Concatenate("Features", HousingRegression.Features)
-                .Append(mlContext.Transforms.Normalize())
                 .AppendCacheCheckpoint(mlContext)
```

Comment:
This was not doing anything, because no column was specified (if I specified the `Feature` column, it alters the test behavior).
Rogan was fine with me removing the line here and below.